resources/hu/cemu.po

```diff
@@ -1740,8 +1934,8 @@ msgid "&H264 API"
 msgstr "&H264 API"
 
 #: MainWindow.cpp
-msgid "&NFP API"
-msgstr "&NFP API"
+msgid "&Graphic pack patches"
+msgstr "&Grafikus csomag-javítások"
```

Comment:
In this string "patches" refers to the ability of Cemu to apply patches (mods) to games through graphic packs. Putting "javítások" in Google Translate yields me "fixes" as main translation, so I just want to be sure you didn't misinterpret this string.

Reply:
Ah, I see. I'll fix it.